### DISCUSSION TITLE
[F] Color conversions provided

### DISCUSSION THREAD

**Discussion by giordifungula** [exceljs/exceljs#2463] [Ideas & Proposals]
## 🚀 Feature Proposal
Color converter for rgb colors to argb for usage on exeljs
A clear and concise description of what the feature is.
Can we use have a list of color conversions or at least a resource provided on the docs on where to go to convert any hex or rgb colors to argb? 
## Motivation
I ran into a issue with a specific color (#002060)  that I want to fill my cell with and the argb value of this color is not easily find-able.
Please outline the motivation for the proposal.
See example of what I would like to do
fgColor: { argb: 'FF00FF00' }, # this is a argb color of green

fgColor: { argb: 'converted collor of hex code 00206' },

**Comment by lee576** [comment]
just try below function

                  // 8位字符颜色先转rgb再转16进制颜色
                  backGroundColor = ((val) => {
                  	val = val.trim()
                  		.toLowerCase(); //去掉前后空格
                  	let color = {};
                  	try {
                  		let argb = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(val);
                  		color.r = parseInt(argb[2], 16);
                  		color.g = parseInt(argb[3], 16);
                  		color.b = parseInt(argb[4], 16);
                  		color.a = parseInt(argb[1], 16) / 255;
                  	} catch (e) {
                  		console.log(e)
                  	}
                  	return "#" + ((1 << 24) + (color.r << 16) + (color.g << 8) + color.b)
                  		.toString(16)
                  		.slice(1);
                  })(cell.style.fill.fgColor.argb)
